--- Crypt-SSleay-0.15/SSLeay.xs.pius	Tue Nov 23 22:19:00 1999
+++ Crypt-SSleay-0.15/SSLeay.xs	Fri Jul 14 00:46:01 2000
@@ -147,7 +147,7 @@
 	   if (n >= 0) {
 	       RETVAL = newSViv(n);
 	   } else {
-	       RETVAL = &sv_undef;
+	       RETVAL = &PL_sv_undef;
            }
 	OUTPUT:
 	   RETVAL
@@ -195,7 +195,7 @@
                buf[offset + n] = '\0';
 	       RETVAL = newSViv(n);
 	   } else {
-	       RETVAL = &sv_undef;
+	       RETVAL = &PL_sv_undef;
            }
 
 	OUTPUT:
