--- Crypt-SSLeay-0.51/SSLeay.xs.orig	2003-05-28 06:55:02.000000000 +0000
+++ Crypt-SSLeay-0.51/SSLeay.xs	2004-05-09 09:04:03.000000000 +0000
@@ -107,7 +107,7 @@
         int rand_bytes_read;
 
 	if(!bNotFirstTime) {
-               SSLeay_add_all_algorithms();
+               SSL_library_init();
                SSL_load_error_strings();
                ERR_load_crypto_strings();
 	       bNotFirstTime = 1;
